projects/cognito-service/src/lib/cognito.service.ts

```diff
@@ -676,6 +676,11 @@ export class CognitoService
   // NOTE: Authentication ----------------------------------------------------------------------
   // -------------------------------------------------------------------------------------------
 
+  public login(provider : string, username ?: string, password ?: string) : Observable<any>
+  {
+    this.authenticateUser(provider, username ? username: null , password ? password: null );
+  }
+  
   public authenticateUser(provider : string, username ?: string, password ?: string) : Observable<any>
```

Comment:
Rename `authenticateUser` to `signIn` :
```suggestion
   public signIn(provider : string, username ?: string, password ?: string) : Observable<any>
```

Reply:
Done! 😄